front/src/types/filter.ts

```diff
@@ -0,0 +1,9 @@
+import { Order, SortOptions } from "@/types/sort";
+
+export type FilterOptions = {
+    sortBy?: SortOptions;
+    sortOrder?: Order;
+    dateTimeFilter?: TimeOptions;
+};
+
+export type TimeOptions = "Last hour" | "Last day" | "Last week";
```

Comment:
Why does this live here, but `sort` lives in another type file?

Reply:
This is because filter is used by the filter feature whereas sort is used by the table feature